generators/templates/Component.tsx.hbs

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+export interface {{pascalCase name}}Props {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+}
+
+const {{pascalCase name}} = ({ testId = 'store-{{kebabCase name}}', children }: {{pascalCase name}}Props) => {
```

Comment:
I commented about children, but I didn't comment on this line also. In order for it to work the prop interface would need to export something that has children on it or we could use the `forwardRef` here, I prefer the latter.
```suggestion
const {{pascalCase name}} = forwardRef<HTMLDivElement, {{pascalCase name}}Props>(function {{pascalCase name}}(
   { testId = 'store-{{kebabCase name}}', children, ...otherProps },
  ref
) {
```